src/OpenSpaceToolkit/Physics/Time/Instant.cpp

```diff
@@ -433,10 +433,10 @@ Instant Instant::DateTime(const time::DateTime& aDateTime, const Scale& aTimeSca
     }
 
     // Limit the year based on the max number of nanoseconds we can store with a UInt64
-    if ((aDateTime.accessDate().getYear() < 1970) || (aDateTime.accessDate().getYear() > 2500))
```

Comment:
Why was it previously `2030`? (wasn't the previous value `2500`?)

Reply:
The previous value was 2030. I don't recall it being 2500. Looking at the git blame, it hasn't been changed in 6 years.

Reply:
Looks like a reviewed a commit diff by mistake, not the PR diff. And one commit went from 2500 to 2554, hence the confusion.

Reply:
Ah gotcha. Yeah Kyle changed it from 2500 to 2554 to match exactly the number of nanoseconds an Uint64 can track when converted to years (rounded down).